Marlin/pins_MELZI_MAKR3D.h

```diff
@@ -2,10 +2,8 @@
  * Melzi with ATmega1284 (MaKr3d version) pin assignments
  */
 
-#define MELZI
-
 #undef MOTHERBOARD
-#define MOTHERBOARD BOARD_SANGUINOLOLU_11
+#define MOTHERBOARD MELZI
```

Comment:
This is better! In general we probably want to stop redefining `MOTHERBOARD` and either use #define'd flags or just check for `MELZI_MAKR3D` in `pins_SANGUINO_11.h` where it checks for `MELZI`.